str0m/src/media/event.rs

```diff
@@ -144,6 +145,16 @@ pub struct MediaData {
     pub meta: Vec<RtpMeta>,
 }
 
+impl MediaData {
+    /// The packet range this was built from.
+    pub(crate) fn packet_range(&self) -> Option<(SeqNo, SeqNo)> {
```

Comment:
It's impossible to create a `MediaData` without any underlying RTP packets. We are making that assumption elsewhere, so we should be consistent and unwrap to avoid `Option` here too.

Reply:
Will action pending outcome on https://github.com/algesten/str0m/pull/64#discussion_r1107334098